packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -105,10 +112,20 @@ export const BookingFields = ({
               optionInput.placeholder = option.inputPlaceholder;
             }
           });
-
           field.options = options.filter(
             (location): location is NonNullable<(typeof options)[number]> => !!location
           );
+
+          field.options.map((option) => {
+            option.label =
+              option.label.search(/^https?:/) !== -1 ? (
+                <a href={option.label} target="_blank">
+                  {getDomain(option.label)}{" "}
+                </a>
+              ) : (
+                option.label
+              );
+          });
```

Comment:
type error in CI

Reply:
@Udit-takkar: Can I change the type for  option.label to `string | HTMLElement`?